### PR TITLE
Batch update

### PR DESCRIPTION
**Pull request by EchterAgo**

### CODE REVIEW

File: _data/cryptoservices.yml [hunk 1]
```diff
@@ -111,10 +111,11 @@ websites:
   url: https://bitit.io/
   img: Bitit.png
   twitter: Bitit_Gift
-  bch: true
+  bch: false
+  bsv: false
   btc: true
   othercrypto: true
-  doc: https://support.bitit.io/hc/en-us/articles/360000566429-How-to-buy-Bitcoin-Cash-BCH-on-Bitit-by-credit-card-cash-and-wire-transfer-
+  text: Bitit used to support BCH, but as of 2019-01-10 stopped supporting it
```
Comment (kenman345) [inline _data/cryptoservices.yml]: due to international use of the site, I think we might want to use fully qualified names of month/dates moving forward

Comment (kenman345) [reply _data/cryptoservices.yml]: @EchterAgo would you mind making this adjustment in the places you used dates in this PR?

Comment (EchterAgo) [reply _data/cryptoservices.yml]: I used the ISO 8601 date format as it is unambigous.

Comment (kenman345) [reply _data/cryptoservices.yml]: thats fine then, I guess i'm just not as aware of that. Thanks for staying on top of what we should use

Comment (kenman345) [reply _data/cryptoservices.yml]: text block needs to be uneder `exceptions` tag

Comment (EchterAgo) [reply _data/cryptoservices.yml]: oh, missed that. will fix it

**Comment by EchterAgo** [comment]
@kenman345 should be good now

**Pull request merged**